lib/atom-scalariform.js

```diff
@@ -5,6 +5,19 @@ var packagePath = function() {
   return atom.packages.getLoadedPackage('atom-scalariform').path;
 }
 
+var projectPath = function(fileToFormatPath) {
+  var ref = atom.project.getDirectories();
+  var projectPath = '';
+  for (var i = 0, len = ref.length; i < len; i++) {
+    var directory = ref[i];
+    if (fileToFormatPath.indexOf(directory.path) > -1) {
```

Comment:
Probably need to do something like 

`fileToFormatPath.substring(0, fileToFormatPath.lastIndexOf('/')) === directory.path`

Otherwise abc/def/Main.scala would match if you also had a directory abc/ghi/Test.scala?